vignettes/sits.Rmd

```diff
@@ -349,6 +349,110 @@ point_cf.tb <- sits_ndvi_arima_filter(point.tb, apply_whit = FALSE)
+
+Using SOM, we can improve the performance of satellite image time series analysis for producing LUCC maps. SOM is an unsupervised neural network algorithm used to map high dimensional space onto low-dimensional space. The structure of SOM is composed by input and output layers. In the input layer contains all time series to be evaluated, also called training data. The output layer consist of a 2-D grid of neurons. Each neuron has a vector of weights representing a pattern of the input data. These neurons are organized into order in which similar pattern are closer to each other in the grid [@Kohonen2001].
+
+To train the network, the neurons are initialized randomly. The vector of weights of neuron $j$,  $w_j=[w_{j1},\ldots,w_{jn} ]$,  must have the same dimension of a input vector of time series samples $x(t)=[x(t)_1,\ldots,x(t)_n ]$. At each training step $t$, a time series sample $x(t)$ is presented to the network to find the neuron whose weight vector has the smaller distance between them. Distance metrics are utilized to compute distance $D_{j}$ between input vector $x(t)$ and each weight vector of neuron, $w_j$ for each neuron j in the output layer. The most part of literature uses Euclidean distance as metric.
+
+$$
+D_{j}=\sum_{i=1}^{n}{\sqrt{(x(t)_i{-}w_{ji})^{2}}}.
+$$
+
+The neuron that contains the shortest distance is declared the winner neuron, defined here as ${b}$, also know as Best Matching Unit (BMU):
+
+
+$$
+ d_{b}= min \left\{D_1,\ldots, D_J \right\}.
+$$
+
+The BMU and its neighborhood must be updated. The weights are adjusted to increase the similarity with input vector.
+The equation for updating the weight vector is given by:
+
+$$
+ w_{j}(t{+}1)=  w_{j}(t){+}\alpha(t)* h_{b,j}(t) [x(t)_i{-}w_{j}(t)],
+$$
+where $\alpha(t)$ is the learning rate, and it must be set as $0<\alpha(t)<1$ and $h_{b,j}(t)$ is a neighbourhood function.
+
+In sits, the package kohonen (referencia) is used as basis to run the SOM, after the training step, to separate the samples in groups is necessary to label the neurons. A cluster can be one neuron or a set of neurons that belongs to same category. In this step, a neuron is named using majority vote technique through of original label given from sample data set. As a result of this, a sample that was associated to a neuron receives the same label of it. In some cases, no samples is associated to a neuron, then, it receives the label "Noclass".
```

Comment:
(referencia)
to run SOM. After the training step, it is necessary to separate the samples in groups in order to label them?
same category or same label?
in some cases, no samples ARE [...] Such samples receive the label noclass.